showcase/app/templates/components/table.hbs

```diff
@@ -21,7 +21,7 @@
     }}
   >
     <:body as |B|>
-      <B.Tr>
+      <B.Tr id={{B.rowIndex}}>
```

Comment:
Maybe you could add an example that displays the rowIndex visually inside a `td`?

Reply:
I though about it, but what would it add? this is not a UI-related thing, is a functional/code thing, and it's already covered in the integration test, which checks that we're actually outputting/yielding the `rowIndex`value. 
So my thought is that it was not the case to add it to the showcase; but if there are strong feelings about it, I can do it.

Reply:
Just a suggestion. I thought it would work for visual regression and just making it more obvious that it exists in the Showcase context. I don't have strong feelings about it.